include/xtensor/xmath.hpp

```diff
@@ -3013,6 +3013,86 @@ namespace detail {
             return cov(eval(stack(xtuple(x, y))));
         }
     }
+
+    /*
+    * convolution mode placeholders
+    */
+    namespace conv_mode
+    {
+        struct valid{};
+        struct full {};
```

Comment:
Is that intentional to no implement the `same` mode?

Reply:
Yeah I figured I could push that implementation off to another pull request because it can be implemented by trimming the full output.